servers/rendering/renderer_rd/shaders/particles.glsl

```diff
@@ -228,6 +228,14 @@ bool emit_subparticle(mat4 p_xform, vec3 p_velocity, vec4 p_color, vec4 p_custom
 	return true;
 }
 
+vec3 normalize_direction(vec3 direction) {
```

Comment:
I think a name like ``safe_normalize()`` may be a bit more descriptive.

Reply:
Agreed, I've changed it.